### PR TITLE
fix: pass enterprise customer uuid to backend when we create a new report configuration

### PR DESCRIPTION
**Pull request by muhammad-ammar**
# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/ReportingConfig/ReportingConfigForm.jsx [hunk 1]
```diff
@@ -122,6 +122,7 @@ class ReportingConfigForm extends React.Component {
       }
     } else {
       // ...or create a new configuration
+      formData.append('enterprise_customer_id', this.props.enterpriseId);
```
Comment (saleem-latif) [inline src/components/ReportingConfig/ReportingConfigForm.jsx:125]: Can/Should we use `config.enterpriseCustomer.uuid` here like on https://github.com/openedx/frontend-app-admin-portal/pull/756/files#diff-d6c597698145cb29bdb34a6ead6524e9808691c30432db9f451bc697e99b9bd4R117

File: src/components/ReportingConfig/ReportingConfigForm.jsx [hunk 2]
```diff
@@ -381,6 +382,7 @@ ReportingConfigForm.defaultProps = {
 };
 
 ReportingConfigForm.propTypes = {
+  enterpriseId: PropTypes.string.isRequired,
```
Comment (saleem-latif) [inline src/components/ReportingConfig/ReportingConfigForm.jsx:385]: nit: I think we should rename the prop to `enterpriseCustomerId` to make it consistent with other places?

**Review by saleem-latif — APPROVED** [review]
Just a couple of suggestions, other than that looks good  👍

File: src/components/ReportingConfig/ReportingConfigForm.jsx (hunk 1 above)
Comment (muhammad-ammar) [reply src/components/ReportingConfig/ReportingConfigForm.jsx:125]: `config` object is available only when we are updating an existing report config

**Pull request merged**